test/getFolderInPath.test.js

```diff
@@ -0,0 +1,43 @@
+const t = require('tap')
```

Comment:
```suggestion
'use strict'

const t = require('tap')
```

I'd also like to make unit test suites sibling files. So this would be `lib/get-folder-in-path.test.js`. We'd want to add a `.taprc.yaml` file that looks like:

```yaml
files:
  - 'index.test.js'
  - 'lib/**/*.test.js'
  - 'test/**/*.test.js'
```